### ISSUE TITLE
Parcel Macro in the REPL

### ISSUE THREAD

**Issue by yfzhe** [parcel-bundler/parcel#9557]
# 🐛 bug report

Parcel Macro and the REPL are both great features! However, it failed when I tried to write macros in the REPL.

For simplification, I use the code from the `getRandomNumber` example of macros: [REPL link](https://repl.parceljs.org/#JTdCJTIyZmlsZXMlMjIlM0ElNUIlNUIlMjIlMkZzcmMlMkZpbmRleC50cyUyMiUyQyU3QiUyMnZhbHVlJTIyJTNBJTIyaW1wb3J0JTIwJTdCJTIwZ2V0UmFuZG9tTnVtYmVyJTIwJTdEJTIwZnJvbSUyMCcuJTJGbWFjcm8uanMnJTIwd2l0aCUyMCU3QiUyMHR5cGUlM0ElMjAlNUMlMjJtYWNybyU1QyUyMiUyMCU3RCUzQiU1Q24lNUNuY29uc29sZS5sb2coZ2V0UmFuZG9tTnVtYmVyKCkpJTNCJTIyJTJDJTIyaXNFbnRyeSUyMiUzQXRydWUlN0QlNUQlMkMlNUIlMjIlMkZzcmMlMkZtYWNyby5qcyUyMiUyQyU3QiUyMnZhbHVlJTIyJTNBJTIyZXhwb3J0JTIwZnVuY3Rpb24lMjBnZXRSYW5kb21OdW1iZXIoKSUyMCU3QiU1Q24lMjAlMjByZXR1cm4lMjBNYXRoLnJhbmRvbSgpJTNCJTVDbiU3RCUyMiU3RCU1RCU1RCUyQyUyMm9wdGlvbnMlMjIlM0ElN0IlMjJlbnRyaWVzJTIyJTNBJTVCJTVEJTJDJTIybWluaWZ5JTIyJTNBZmFsc2UlMkMlMjJzY29wZUhvaXN0JTIyJTNBdHJ1ZSUyQyUyMnNvdXJjZU1hcHMlMjIlM0FmYWxzZSUyQyUyMnB1YmxpY1VybCUyMiUzQSUyMiUyRl9fcmVwbF9kaXN0JTIyJTJDJTIydGFyZ2V0VHlwZSUyMiUzQSUyMmJyb3dzZXJzJTIyJTJDJTIydGFyZ2V0RW52JTIyJTNBbnVsbCUyQyUyMm91dHB1dEZvcm1hdCUyMiUzQW51bGwlMkMlMjJobXIlMjIlM0FmYWxzZSUyQyUyMm1vZGUlMjIlM0ElMjJwcm9kdWN0aW9uJTIyJTJDJTIycmVuZGVyR3JhcGhzJTIyJTNBZmFsc2UlMkMlMjJ2aWV3U291cmNlbWFwcyUyMiUzQWZhbHNlJTJDJTIyZGVwZW5kZW5jaWVzJTIyJTNBJTVCJTVEJTJDJTIybnVtV29ya2VycyUyMiUzQTAlN0QlMkMlMjJ1c2VUYWJzJTIyJTNBZmFsc2UlMkMlMjJicm93c2VyQ29sbGFwc2VkJTIyJTNBJTVCJTVEJTJDJTIydmlld3MlMjIlM0ElNUIlMjIlMkZzcmMlMkZpbmRleC50cyUyMiUyQyUyMiUyRnNyYyUyRm1hY3JvLmpzJTIyJTVEJTJDJTIyY3VycmVudFZpZXclMjIlM0EyJTdE)

## 🤔 Expected Behavior

```javascript
console.log(/* a random number here */);
```

## 😯 Current Behavior

``` javascript
(function () {
function $fe2e15112bb76575$export$eab19c48dcb6310d() {
    return Math.random();
}


console.log((0, $fe2e15112bb76575$export$eab19c48dcb6310d)());

})();
```

## 🌍 Your Environment

// It says "Based on commit [2059029ee9](https://github.com/parcel-bundler/parcel/commits/2059029ee91e5f03a273b0954d3e629d7375f986)" in the REPL.

**Comment by devongovett** [comment]
Yeah I think it doesn't work in the wasm build yet. We'd need to do something like lightningcss using the asyncify transform to support async functions called from rust. https://github.com/parcel-bundler/lightningcss/blob/master/wasm/async.mjs
